src/maker.rs

```diff
@@ -231,6 +273,10 @@ pub struct PublishOrders {
 #[error("Rate not available when trying to create new {0} order.")]
 pub struct RateNotAvailable(Position);
 
+#[derive(Debug, Clone, thiserror::Error)]
+#[error("{0} balance not available.")]
+pub struct BalanceNotAvailable(String);
```

Comment:
```suggestion
pub struct BalanceNotAvailable(Symbol);
```
Please use a type.

Reply:
I think we could reuse the symbol introduced by https://github.com/coblox/nectar/pull/48/files here? 
I kept it as string for now, because I treat it as a `const` of the module atm. @D4nte would you mind creating a follow up for this?

Reply:
Hum no, I would not re-use `history::Symbol`. Nothing in `history` is to be exposed. Only things that are exposed in `history` are:

- `History::new`
- `History::write`
- `impl From<..::Swap< for history::Swap` (to be done)

Reply:
introduced separate enum